pkg/operator/k8sutil/resources.go

```diff
@@ -147,6 +147,11 @@ func (info *OwnerInfo) GetUID() types.UID {
 	return info.owner.GetUID()
 }
 
+// GetOwnerReference gets the owner references of the owner
+func (info *OwnerInfo) GetOwnerReference() *metav1.OwnerReference {
```

Comment:
"Get" implies it requires more work than just returning a field.
```suggestion
func (info *OwnerInfo) OwnerReference() *metav1.OwnerReference {
```